### PR TITLE
Add section about pragma marks

### PR DESCRIPTION
**Pull request by zenangst**

### CODE REVIEW

**Comment by zenangst** [comment]
This touches on this issue - https://github.com/hyperoslo/iOS-playbook/issues/18

File: STYLE_GUIDELINES.md [hunk 1]
```diff
@@ -259,6 +259,32 @@ When they are needed, comments should be used to explain **why** a particular pi
...
+Private and custom delegate methods are added at the bottom.
+
+```objc
+#pragma mark - Initializers
+
+#pragma mark - Getters
+
+#pragma mark - Setters
+
+#pragma mark - View Life Cycle
```
Comment (NSElvis) [inline STYLE_GUIDELINES.md]: Might be better to have `View Life cycle` (lowercase for cycle)

**Comment by NSElvis** [comment]
besides the commented point looks good :+1:

**Comment by zenangst** [comment]
@NSElvis fixed

**Comment by timkurvers** [comment]
Why would `Life` remain with a capital?

**Comment by NSElvis** [comment]
@timkurvers you are right.

:bow: :thought_balloon:

**Comment by NSElvis** [comment]
@zenangst @timkurvers https://github.com/hyperoslo/iOS-playbook/pull/55

**Pull request merged**